### PR TITLE
[WIP] feat(migration): upgrade to symfony 4

### PR DESCRIPTION
**Pull request by nayluge**
Based on the work of @Shine-neko and @hhamon.

Some things need to be checked before merging and deployed :

- [ ]  Deployment needs to be changed
- [ ]  The old parameters.yml variables have to be moved to env vars.
- [ ] The admin of article's edition is not clear. Not sure it works properly.

### CODE REVIEW

File: src/Controller/BlogController.php [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        $this->tagManager = $tagManager;
+        $this->doctrine = $doctrine;
+    }
+
+    /**
+     * @Route("/", name="blog_home")
+     */
+    public function indexAction(Request $request)
+    {
+        $query = $this->getDoctrine()->getRepository('App:Article')->getQuery();
```
Comment (Shine-neko) [inline src/Controller/BlogController.php]: Why not create service for this ?

**Comment by Nek-** [comment]
There's no security working x).

File: src/Controller/BlogController.php (hunk 1 above)
Comment (Nek-) [reply src/Controller/BlogController.php]: First I see nothing wrong with it. But also it was like this before: https://github.com/afsy/website/blob/master/src/Afsy/Bundle/FrontBundle/Controller/BlogController.php#L27

**Comment by nayluge** [comment]
@Nek- : fix applied.

The yarn.lock file is commited ;)

File: assets/js/app.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+// any CSS you require will output into a single css file (app.css in this case)
+require('../css/reset.css');
+require('../css/global.css');
+require('../css/layout.css');
+require('../css/app.css');
+require('../css/grid.css');
+require('../css/icons.css');
+require('../css/avent.css');
+
+const $ = require('jquery');
```
Comment (stof) [inline assets/js/app.js]: this (local) variable is unused

**Pull request merged**